pkg/registration/helpers/helpers.go

```diff
@@ -158,3 +159,12 @@ func IsCSRSupported(nativeClient kubernetes.Interface) (bool, bool, error) {
 	}
 	return v1CSRSupported, v1beta1CSRSupported, nil
 }
+
+func IsEksArnWellFormed(eksArn string) bool {
```

Comment:
add some comment for this func, it will be easier to understand what is the format for the arn.